FFXIVClientStructs/FFXIV/Component/GUI/AtkComponentBase.cs

```diff
@@ -62,6 +62,7 @@ public unsafe partial struct AtkComponentBase {
     [VirtualFunction(5)]
     public partial void OnUldUpdate();
 
+    // TODO: return void
     [VirtualFunction(10)]
     public partial void* SetEnabledState(bool enabled);
```

Comment:
If this is wrong, shouldn't it get fixed even if it's a "breaking change"?
Returning things when you shouldn't is likely to cause problems.

Reply:
Dunno. Thats a question for @pohky, I guess.

Reply:
This is yes wrong and yes breaking, but looking at IDA and how it resolves how to return something based on the return type indicates that it's not major enough until a larger breaking change is required to change this also.